src/ServerRequestFactory.php

```diff
@@ -455,4 +457,23 @@ private static function normalizeNestedFileSpec(array $files = [])
         }
         return $normalizedFiles;
     }
+
+    /**
+     * Return HTTP protocol version (X.Y)
+     *
+     * @param array $server
+     * @return string
+     */
+    public static function marshalProtocolVersion(array $server)
```

Comment:
This should also be private, for consistency with the other methods. I'll make that change on merge.